webapp/channels/src/components/admin_console/system_users/system_users_search/index.tsx

```diff
@@ -4,22 +4,26 @@
 import type {ChangeEvent} from 'react';
 import React, {useRef, useState} from 'react';
 import {useIntl} from 'react-intl';
-import {useDispatch, useSelector} from 'react-redux';
+import {useDispatch} from 'react-redux';
 
 import {setAdminConsoleUsersManagementTableProperties} from 'actions/views/admin';
-import {getAdminConsoleUserManagementTableProperties} from 'selectors/views/admin';
 
 import Input from 'components/widgets/inputs/input/input';
 
+import type {AdminConsoleUserManagementTableProperties} from 'types/store/views';
+
 import './system_users_search.scss';
 
-export function SystemUsersSearch() {
+type Props = {
+    searchTerm: AdminConsoleUserManagementTableProperties['searchTerm'];
+}
+
+export function SystemUsersSearch(props: Props) {
     const {formatMessage} = useIntl();
     const dispatch = useDispatch();
     const timeout = useRef<NodeJS.Timeout>();
 
-    const initialValue = useSelector(getAdminConsoleUserManagementTableProperties).searchTerm;
-    const [inputValue, setInputValue] = useState(initialValue);
+    const [inputValue, setInputValue] = useState(props.searchTerm);
```

Comment:
Is there a reason we are prop drilling this instead of grabbing it from redux? Does it matter?